app/src/main/cpp/HandModels.h

```diff
@@ -0,0 +1,40 @@
+/* -*- Mode: C++; tab-width: 20; indent-tabs-mode: nil; c-basic-offset: 2 -*-
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef VRBROWSER_HANDMODELS_H
+#define VRBROWSER_HANDMODELS_H
```

Comment:
Instead of doing this (which is old fashioned) please simply use `#pragma once`

Reply:
Yes we actually use it quite a lot

Reply:
Ok, will fix that.